test/test_routes.py

```diff
@@ -720,25 +726,6 @@ def test_orjson_serialization():
     demo.close()
 
 
-def test_file_route_does_not_allow_dot_paths(tmp_path):
```

Comment:
We don't need a dedicated test suite for this since this is no longer a special case (we simply block all files in the working directory)